tools/bin_to_c.c

```diff
@@ -0,0 +1,143 @@
+//
+// bin_to_c - convert binary data into c-compatible data tables
+//
+// Written by Larry Bank
+// Copyright (c) 2009 BitBank Software, Inc.
+// Change history
+// 2/1/09 - Started the project
+// 11/6/15 - converted to Linux
+// 7/29/20 - tailored to Arduino FLASH data output
+//
```

Comment:
* Use C-style comments.
* Put the original copyright notice: https://github.com/bitbank2/bin_to_c/blob/master/LICENSE and keep "Written by Larry Bank" and "Copyright (c) 2015 BitBank Software, Inc."
* Remove the change history for short.